ragna/embedding_models/_embedding.py

```diff
@@ -0,0 +1,14 @@
+from ragna.core import Chunk, Embedding, EmbeddingModel
+
+
+class MiniLML6v2(EmbeddingModel):
+    def __init__(self):
+        super().__init__()
+        from chromadb.utils import embedding_functions
+        self.model = embedding_functions.DefaultEmbeddingFunction()
+
+    def embed_chunks(self, chunks: list[Chunk]) -> list[Embedding]:
+        return [Embedding(embed_chunk[0], embed_chunk[1]) for embed_chunk in zip(self.embed_text([chunk.text for chunk in chunks]), chunks)]
+
+    def embed_text(self, text: list[str]) -> list[float]:
+        return self.model(text)
```

Comment:
The input here is annotated as `list[str]` and it is passed in this way through `embed_chunks`. However in `Chat.answer` we just pass the string. One of them cannot be right. IMO we should just take `str` here and refactor the `embed_chunks` method.

Reply:
I would rather make it optional and handle the case of `str` being passed, because batching these should in theory make it much faster.

Reply:
Sorry for my ignorance here: are you saying that passing `list[str]` runs all items of that list in parallel? If so, I totally agree that we should do this. Can we assert this assumption though? A quick benchmark will do.